test/integration/user_checks_out_from_cart_test.rb

```diff
@@ -35,6 +41,12 @@ class UserChecksOutFromCartTest < ActionDispatch::IntegrationTest
     user = create(:user)
     ApplicationController.any_instance.stubs(:current_user).returns(user)
 
+    StripeMock.start
+    customer = Stripe::Customer.create({
+      email: user.email_address,
+      card: '4242424242424242'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.